server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanQuestions.ts

```diff
@@ -0,0 +1,96 @@
+import type { ContingencyPlanQuestionId, ContingencyPlanQuestionsBody, TaskListErrors } from '@approved-premises/ui'
+import { Page } from '../../../utils/decorators'
+
+import TasklistPage from '../../../tasklistPage'
+
+const questions: ContingencyPlanQuestionsRecord = {
+  noReturn: {
+    question: 'If the person does not return to the AP for curfew, what actions should be taken?',
+    hint: 'List all actions and agencies involved. Identify who is responsible for each section.',
+    error: 'You must detail the actions that should be taken if the person does not return to the AP for curfew',
+  },
+  placementWithdrawn: {
+    question: "If the person's placement needs to be withdrawn out of hours, what actions should be taken?",
+    hint: `Placements can be withdrawn due to safety, security or behavioural issues.
+    List all actions and agencies involved. Identify who is responsible for each section.`,
+    error:
+      "You must detail any actions that should be taken if the person's placement needs to be withdrawn out of hours",
+  },
+  victimConsiderations: {
+    question: 'Provide any victim considerations that the AP need to be aware of when out of hours',
+    error: 'You must detail any victim considerations that the AP need to be aware of when out of hours',
+  },
+  unsuitableAddresses: {
+    question:
+      'In the event of an out of hours placement withdrawal, provide any unsuitable addresses that the person cannot reside at',
+    hint: 'List all actions and agencies involved. Identify who is responsible for each section.',
+    error: 'You must detail any unsuitable addresses that the person cannot reside at',
+  },
+  suitableAddresses: {
+    question:
+      'In the event of an out of hours placement withdrawal, provide alternative suitable addresses that the person can reside at',
+    hint: 'For situations where a placement is withdrawn and recall action is not taken, the person will need to be directed to alternative places to reside in',
+    error: 'You must detail alternative suitable addresses that the person can reside at',
+  },
+  breachInformation: {
+    question: 'In the event of a breach, provide any further information to support Out of Hours (OoH) decision making',
+    hint: 'For example, include any required actions or views from the probation practicioner regarding OoH recall following curfew breach',
+    error: 'You must detail any further information to support OoH decision making',
+  },
+  otherConsiderations: {
+    question: 'Are there any other considerations?',
+    error: 'You must detail any other considerations',
+  },
+} as const
+
+type ContingencyPlanQuestion = {
+  question: string
+  hint?: string
+  error: string
+}
+
+type ContingencyPlanQuestionsRecord = Record<ContingencyPlanQuestionId, ContingencyPlanQuestion>
```

Comment:
Is it intentional that these are duplicating what's in the UI types file?

Reply:
Nope, have fixed